### PR TITLE
Feat: 🚀 New addon certification-manager, version: 1.13.1

### PR DESCRIPTION
**Pull request by VishwajitNagulkar**
## what
* Created Addon for certification manager

## why
* Certification manager addon required for github-runner controller

### CODE REVIEW

**Review by d4kverma — APPROVED** [review]
LGTM

File: addons/cert-manager/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Below terraform script shows how to use Certification Manager Terraform Addon, A complete example is also given [here](https://github.com/clouddrove/terraform-helm-eks-addons/blob/master/_examples/complete/main.tf).
+```hcl
+module "addons" {
+  source  = "clouddrove/eks-addons/aws"
+  version = "1.0.0"
+  
+  depends_on       = [module.eks.cluster_id]
+  eks_cluster_name = module.eks.cluster_name
+
+  certification_manager   = true
```
Comment (h1manshu98) [inline addons/cert-manager/README.md:16]: - `certification_manager` will be available to use in next release. Please update version to `"0.1.0"`
- also update `depends_on = [module.eks]`

File: addons/cert-manager/locals.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    replace                    = try(var.certification_manager_extra_configs.replace, "false")
+  }
+
+  certification_manager_extra_configs = var.certification_manager_extra_configs
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config,
+    local.certification_manager_extra_configs
+  )
```
Comment (h1manshu98) [inline addons/cert-manager/locals.tf:43]: Line-37 and Line-42 can be removed.

File: addons/cert-manager/main.tf [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  depends_on = [kubernetes_namespace_v1.this]
+}
+
+resource "kubernetes_namespace_v1" "this" {
+  count = try(local.helm_config["create_namespace"], true) && local.helm_config["namespace"] != "cert-manager" ? 1 : 0
+
+  metadata {
+    name = local.helm_config["namespace"]
+  }
+}
```
Comment (h1manshu98) [inline addons/cert-manager/main.tf:17]: resource `kubernetes_namespace_v1` can be removed, as namespace creation is controller by [create_namespace](https://github.com/clouddrove/terraform-aws-eks-addons/blob/0c6885f00e7225355bf16d3dc29c88fc3504831c/addons/cert-manager/locals.tf#L10) variable. Remove Line-8 `depends_on` as well.

**Review by h1manshu98 — APPROVED** [review]
LGTM!

**Review by d4kverma — APPROVED** [review]
LGTM

**Review by AshutoshAM2002 — APPROVED** [review]
LGTM

**Pull request merged**